src/Components/SampleEditor/MarkdownViewer/MarkdownViewer.tsx

```diff
@@ -1,10 +1,11 @@
+
 /*---------------------------------------------------------------------------------------------
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 import * as React from "react";
 import Markdown from "markdown-to-jsx";
-import { SampleLink } from "../MarkdownComponents/Link/SampleLink";
+import { Link } from "../MarkdownComponents/Link/OldLink";
```

Comment:
Does naming the file "ReadMeLink" make more sense than "OldLink"?

Reply:
Yeah i'll can change it. I called it 'OldLink' because its the Old Link.tsx file. The new changes that Stefan introduced with walkthroughs didn't agree with the current Readme.